instrumentation/thrift-0.14.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/thrift/ClientOutProtocolWrapper.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.thrift;
+
+import static io.opentelemetry.javaagent.instrumentation.thrift.ThriftSingletons.clientInstrumenter;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import javax.annotation.Nullable;
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TMessage;
+import org.apache.thrift.protocol.TProtocol;
+
+@SuppressWarnings({"serial"})
```

Comment:
Is this `@SuppressWarnings` needed, this class does not seem to be serializable?